compiler/src/iree/compiler/Codegen/Common/CPU/CPULowerToUKernels.cpp

```diff
@@ -12,6 +12,7 @@
 #include "iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenOps.h"
 #include "iree/compiler/Codegen/Dialect/Codegen/IR/UKernelOps.h"
 #include "iree/compiler/Codegen/Utils/Utils.h"
+#include "iree/compiler/Dialect/Encoding/IR/EncodingOps.h"
 #include "iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.h"
```

Comment:
I think we don't need LinalgExt dep in this file, and we can drop the LinalgExt dep from `Common/CPU`.